.asf.yaml

```diff
@@ -67,16 +67,16 @@ github:
           - cypress-matrix (2, chrome)
           - cypress-matrix (3, chrome)
           - frontend-build
-          - pre-commit (3.9)
-          - python-lint (3.9)
-          - test-mysql (3.9)
-          - test-postgres (3.9)
-          - test-postgres (3.10)
-          - test-sqlite (3.9)
+#          - pre-commit (3.9)
```

Comment:
@EugeneTorap why not simply remove as opposed to comment out. Additionally why is line #74 commented out?